satellite/inspector/inspector.go

```diff
@@ -0,0 +1,184 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package inspector
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
```

Comment:
maybe you already know this because travis, but this space is making the linter fail.